common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/OneForOneBlockFetcher.java

```diff
@@ -81,7 +81,6 @@ public OneForOneBlockFetcher(
       TransportConf transportConf,
       DownloadFileManager downloadFileManager) {
     this.client = client;
-    this.blockIds = blockIds;
```

Comment:
Do we need to remove this from the constructor, line 78 and line 69?

Reply:
Or, we cannot change it because it's a public class?

Reply:
The blockIds will be used to create OpenBlocks or ShuffleFetchBlocks later in contructor.